staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics/metrics.go

```diff
@@ -146,6 +146,18 @@ var (
 		},
 		[]string{"resource"},
 	)
+
+	WatchListLatencies = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "watch_list_duration_seconds",
+			Help:           "Response latency distribution in seconds for watch list requests broken by group, resource.",
+			Buckets:        []float64{0.5, 1.0, 2, 5, 8, 10, 15, 20, 30, 45, 60},
```

Comment:
Bucket don't necessarily need to match to be able to compare to the kube-apiserver metrics, but isn't `0.5` second quite a high for resource-scoped watch list requests?

Reply:
maybe @wojtek-t can comment on that. I don't have experience with actual numbers.